tests/components/fritz/test_init.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for AVM Fritz!Box."""
+from unittest.mock import patch
+
+from fritzconnection.core.exceptions import FritzSecurityError
+import pytest
+
+from homeassistant.components.device_tracker.const import (
+    CONF_CONSIDER_HOME,
+    DEFAULT_CONSIDER_HOME,
+)
+from homeassistant.components.fritz.const import DOMAIN, FRITZ_EXCEPTIONS
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .const import MOCK_USER_DATA
+
+from tests.common import MockConfigEntry
+
+
+async def test_setup(hass: HomeAssistant, fc_class_mock, fh_class_mock):
+    """Test setup and unload of Fritz!Tools."""
+
+    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_USER_DATA)
+    entry.add_to_hass(hass)
+
+    assert await async_setup_component(hass, DOMAIN, {})
+    await hass.async_block_till_done()
+    assert entry.state == ConfigEntryState.LOADED
```

Comment:
These lines are present in many tests, maybe have a fixture for default setup of the entry?

Reply:
tbh, not in this PR 🙈